automated-comments/elixir/two-fer/raise_fn_clause_error.md

```diff
@@ -0,0 +1,18 @@
+<!--
+This error is raised when the submission includes a call to
+`raise FunctionClauseError` rather than having elixir fail in
```

Comment:
I think this can be made a generic comment, as `FunctionClauseError` should be rarely raised by the developer, not only not in `two-fer`.

Reply:
moved to solution rather than two-fer